src/modules/blockchainApplication/explore/components/BlockchainApplicationStatistics/index.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Box from 'src/theme/box';
+import BoxHeader from 'src/theme/box/header';
+import BoxContent from 'src/theme/box/content';
+import { DoughnutChart } from 'src/modules/common/components/charts';
+import TokenAmount from '@token/fungible/components/tokenAmount';
+import { tokenMap } from '@token/fungible/consts/tokens';
+import Tooltip from 'src/theme/Tooltip';
+import Icon from 'src/theme/Icon';
+import { useTheme } from 'src/theme/Theme';
+import { getColorPalette } from 'src/modules/common/components/charts/chartOptions';
+import styles from './blockchainApplicationStatistics.css';
+
+const BlockchainApplicationStatistics = ({ apps, statistics }) => {
+  const { t } = useTranslation();
+  const colorPalette = getColorPalette(useTheme());
+
+  const doughnutChartData = {
+    labels: [
+      t('Registered'),
+      t('Active'),
+      t('Terminated'),
+    ],
+    datasets: [
+      {
+        backgroundColor: [colorPalette[1], colorPalette[0], colorPalette[2]],
+        data: apps.data.reduce((acc, stats) => {
```

Comment:
why? the api response it will give us the data

```
{  
"data":  {    
   "registered": 2503,
    "active": 2328, 
   "terminated": 35,
    “totalSupplyLSK”: “5000000”, 
   “stakedLSK”: “3000000”, 
   “inflationRate”: “4.50”, 
 }, 
  "meta": {}, 
 "links": {}
}

Reply:
There are 2 callls, 1 for apps list and other for apps. We have to iterate the apps list to get the status and generate the status map. As far as I understood

Reply:
please go over the end point and check it one more time, 

api is providing the application count based on the status
`
"registered": 2503,
    "active": 2328, 
   "terminated": 35,
   `